app/views/layouts/application_react.html.erb

```diff
@@ -0,0 +1,21 @@
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta data-hook="head-js">
+    <title>React App</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no">
+    <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track' => true %>
```

Comment:
we said that we will use styled_components so probably we don't need all the mess be loaded here :smile 
just load bootstrap css